gatling-jms/src/main/scala/io/gatling/jms/request/JmsAttributes.scala

```diff
@@ -30,7 +30,7 @@ import io.gatling.core.session.Expression
 case class JmsAttributes(
   requestName:       String,
   destination:       JmsDestination,
-  replyDestination:  JmsDestination,
+  replyDestination:  Option[JmsDestination],
```

Comment:
Please don't duplicate. I'd say that we could move replyDestination out of JmsAttribute and directly store it into JmsReqReplyBuilder